### PR TITLE
Update testing.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no 
| New feature?  | yes/no 
| BC breaks?    | no
| Deprecations? | yes/no
| Tickets       | fixes #... 
| License       | MIT
| Doc PR        | api-platform/docs#...

### CODE REVIEW

File: distribution/testing.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ You may also be interested in these alternative testing tools (not included in t
 * [Behat](http://behat.org/en/latest/) and its [Behatch extension](https://github.com/Behatch/contexts), a
   [Behavior-Driven development](https://en.wikipedia.org/wiki/Behavior-driven_development) framework to write the API
   specification as user stories and in natural language then execute these scenarios against the application to validate
-  its behavior;
+  it's behavior;
```
Comment (antograssiot) [inline distribution/testing.md]: This was correct before i guess

Comment (sakshamtaneja21) [reply distribution/testing.md]: as per you 
as per mine this should be changed

**Comment by sakshamtaneja21** [comment]
how can I get an invite of your GitHub organization as a member

File: distribution/testing.md (hunk 1 above)
Comment (dunglas) [reply distribution/testing.md]: as per the English grammar...

**Comment by dunglas** [comment]
You can't. Please stop spamming this repository.

**Pull request closed without merging**